apps/web/pages/product/[slug].vue

```diff
@@ -39,21 +39,25 @@ const { setSingleItemMeta } = useStructuredData();
 const route = useRoute();
 const { selectVariation } = useProducts();
 const localePath = useLocalePath();
+
 definePageMeta({
   layout: false,
 });
 
 const { productParams, productId } = createProductParams(route.params);
 const { data: product, fetchProduct, setTitle, generateBreadcrumbs, breadcrumbs } = useProduct(productId);
 const { data: productReviewAverage, fetchProductReviewAverage } = useProductReviewAverage(productId);
+const { fetchProductReviews, data: productReviews } = useProductReviews(Number(productId));
+await Promise.all([
+  fetchProduct(productParams),
+  fetchProductReviewAverage(Number(productId)),
+  fetchProductReviews(Number(productId)),
```

Comment:
We might have to add this only if its on ssr. On client, I would think it's only necessary to load it once you want to see the reviews.

Reply:
if request was exectued on server, then you have the data on client for what i know,
i think there is no win by doing so